pkg/solana/chainreader/event_read_binding.go

```diff
@@ -521,6 +521,10 @@ func valueForPath(from reflect.Value, itemType string) (any, error) {
 
 	switch from.Kind() {
 	case reflect.Pointer:
+		if from.IsNil() {
+			from = reflect.New(from.Type().Elem())
+		}
+
```

Comment:
Ah, yes... this is one thing I was thinking would be necessary for these tests to pass!